pkg/operator/ceph/csi/spec.go

```diff
@@ -417,7 +417,7 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 		// apply resource request and limit to rbd provisioner containers
 		applyResourcesToContainers(clientset, rbdProvisionerResource, &rbdProvisionerDeployment.Spec.Template.Spec)
 		k8sutil.SetOwnerRef(&rbdProvisionerDeployment.ObjectMeta, ownerRef)
-		antiAffinity := getPodAntiAffinity("app", "csi-rbdplugin-provisioner")
+		antiAffinity := GetPodAntiAffinity("app", "csi-rbdplugin-provisioner")
```

Comment:
Should use const for "csi-rbdplugin-provisioner". This const probably already exists in the CSI code.

Reply:
I just needed the GetPodAntiAffinity function exported. This is unrelated to admission-controllers PR. We could include the fixes for CSI code in another PR maybe ?